dome/GUI_tutorial.md

```diff
@@ -0,0 +1,116 @@
+# Understanding the GUI
+
+This guide is an introduction to the Ignition Graphical User Interface (GUI).
+
+## Prerequisites
+
+Before beginning, you should have [Ignition installed](/docs/dome/install).
+If you installed [from source](/docs/dome/install)
+you will need to [source](/docs/dome/install_ubuntu_src#using-the-workspace) the workspace;
```

Comment:
```suggestion
Before beginning, you should have [Ignition installed](install.md).
If you installed [from source](install.md)
you will need to [source](install_ubuntu_src.md#using-the-workspace) the workspace;
```

It looks like we are missing the file extensions in order to make these links work.

Also, since all of these files are in the same directory, would it be best to omit the path since that assumes the current directory?